### ISSUE TITLE
v1.18.10 : Operations parsing in dependencies

### ISSUE THREAD

**Issue by maitredede** [swaggo/swag#1451]
**Describe the bug**

With update 1.18.19 #1432, our swag generation goes empty, since all our operations are defined in sub-packages.

**To Reproduce**

1. Create a package with `cmd/api` with files `main.go` (with gin router init and run) and `swag.go` (with `go:generate` and swag comments`
2. Create subpackages `pkg/api/user`, `pkg/api/todolist`, with gin handlers and swaggo comments.
3. `go generate ./...`

**Expected behavior**

Generated docs should contains oeprations, but it does not.

**Your swag version**

v1.18.10

**Your go version**

go1.19.5 linux/amd64

**Additional context**

A solution proposal : add a cli flag that allows to revert #1432 behavior (keeps #1432 behavior by default).

**Comment by Jictyvoo** [comment]
I have the same problem here, as all my operations are defined in another modules that are inside my project monorepo

**Comment by bastianwegge** [comment]
@sdghchj we really appreciate your work, swaggo has done a great job for our swagger specs. Is there any chance to 
enable dependency parsing with the flag that maitredede proposed? We're generating >500 routes into a `generated` package and mounting these inside of our app that generates the swagger specs. #1432 is a dealbreaker for us to use `swaggo`.

**Comment by sdghchj** [comment]
Hi，everyone，sorry to have made this trouble to you。But it may not be a breakup，just to change you swag cmd to fix, as swag has a cmd line flag -d to set multiple directories separated by comma.
for example:
swag init -g main.go -d cmd/api,pkg/api

**Comment by sdghchj** [comment]
it is suggested that if you want to parse apis in some external packges，just explicitly include them with flag -d

**Comment by matdurand** [comment]
The problem here is that it seems like a major breaking change to stop including operations in dependencies, and this was included in a revision release. Very confusing for anyone just casually updating their dependencies.

**Comment by arminbhy** [comment]
> Hi，everyone，sorry to have made this trouble to you。But it may not be a breakup，just to change you swag cmd to fix, as swag has a cmd line flag -d to set multiple directories separated by comma. for example: swag init -g main.go -d cmd/api,pkg/api

Seems like with the change here the --pd is not working anymore, assume it would if you vendor the dependencies and include the vendor path in the -d but not if you are not vendoring.

**Comment by Jictyvoo** [comment]
> > Hi，everyone，sorry to have made this trouble to you。But it may not be a breakup，just to change you swag cmd to fix, as swag has a cmd line flag -d to set multiple directories separated by comma. for example: swag init -g main.go -d cmd/api,pkg/api
> 
> Seems like with the change here the --pd is not working anymore, assume it would if you vendor the dependencies and include the vendor path in the -d but not if you are not vendoring.

Yes, tried here with new version and still have this same behavior, deleting all my swagger docs, but it works on version `v1.8.9`

**Comment by apawelec** [comment]
Is there any progress on this? Due to this behaviour I stuck on v1.8.9.

**Comment by terev** [comment]
Also stuck on v1.8.9. Very curious for the real reason this feature was removed. Especially since this was done in a patch release. Also it kinda seems like the maintiners are against adding it back?
